doc/manual/strings.rst

```diff
@@ -230,12 +230,12 @@ a normal value:
 Using an index less than 1 or greater than ``end`` raises an error::
 
     julia> str[0]
-    ERROR: BoundsError()
-     in getindex at /Users/sabae/src/julia/usr/lib/julia/sys.dylib (repeats 2 times)
+    ERROR: BoundsError: attempt to access 14-element Array{UInt8,1} at index [0]
```

Comment:
Not realted to the PR but I wonder if we can make this error message more clear when indexing into a string.